package.json

```diff
@@ -223,14 +223,14 @@
     "@keystonehq/metamask-airgapped-keyring": "^0.13.1",
     "@lavamoat/snow": "^1.5.0",
     "@material-ui/core": "^4.11.0",
-    "@metamask-institutional/custody-controller": "^0.2.10",
-    "@metamask-institutional/custody-keyring": "^0.0.27",
```

Comment:
@zone-live The cherry-picked PR updated the custody keyring to v1.0.3, but this PR does not. Is this intended?

Reply:
Yeah, the cherry-picked PR didn't think we would go back to 11.1 😆 so I had to decrease those two packages versions to be the same ones we have in 11.1.

Reply:
Meaning, the original PR just had the most up to date versions because it went to be merged against `develop` and since develop has the latest changes, that works.
But since this PR is going against `master` and doesn't have the latest changes we have in develop, those two packages versions have to stay the same.

Don't know if this answered the question 😅

Reply:
That makes sense. I just want to make sure that the issues you need to fix don't rely on v1.0.3 of `custody-keyring`